pynucastro/networks/tests/test_python_net.py

```diff
@@ -6,21 +6,51 @@
 
 class TestPythonNetwork:
     @pytest.fixture(scope="class")
-    def rate(self):
+    def rate1(self):
         return rates.ReacLibRate("c13-pg-n14-nacr")
 
-    def test_ydot_string(self, rate):
-        ydot = rate.ydot_string_py()
-        assert ydot in ("rho*Y[jc13]*Y[jp]*rate_eval.p_c13__n14",
-                        "rho*Y[jp]*Y[jc13]*rate_eval.p_c13__n14")
+    @pytest.fixture(scope="class")
+    def rate2(self):
+        return rates.ReacLibRate("he4-pphe3-he3-nacr")
+
+    @pytest.fixture(scope="class")
+    def rate3(self):
+        return rates.ReacLibRate("he4-npahe3-li7-mafo")
+
+    @pytest.fixture(scope="class")
+    def rate4(self):
+        return rates.ReacLibRate("p-p-d-ec")
+
+    def test_ydot_string(self, rate1, rate2, rate3, rate4):
+        ydot1 = rate1.ydot_string_py()
+        assert ydot1 in ("rho*Y[jc13]*Y[jp]*rate_eval.p_c13__n14",
+                         "rho*Y[jp]*Y[jc13]*rate_eval.p_c13__n14")
```

Comment:
The reactants are sorted, so this shouldn't be necessary anymore
```suggestion
        assert ydot1 == "rho*Y[jp]*Y[jc13]*rate_eval.p_c13__n14"
```